diesel/src/pg/types/mod.rs

```diff
@@ -11,4 +12,6 @@ pub mod sql_types {
     pub type SmallSerial = ::types::SmallInt;
     pub type Serial = ::types::Integer;
     pub type BigSerial = ::types::BigInt;
+    pub type Uuid = super::uuid::uuid::Uuid;
+    pub type Bytea = ::types::Binary;
```

Comment:
Ah, I see the problem. Our schema inference won't actually work just yet with this...